src/ngraph/graph_util.cpp

```diff
@@ -243,6 +243,15 @@ std::list<std::shared_ptr<ngraph::Node>>
             }
             auto cloned_node = node->copy_with_new_args(cloned_args);
 
+            // Copy op annotations.
+            if (auto cloned_op = dynamic_pointer_cast<ngraph::op::Op>(cloned_node))
+            {
+                if (auto op_annotations = cloned_op->get_op_annotations())
+                {
+                    cloned_op->set_op_annotations(op_annotations->clone());
+                }
+            }
+
```

Comment:
Moved the copy of op annotations to this `clone_nodes` utility. I guess it makes more sense since we are also copying control dependencies (below). If it doesn't, I could set the annotation information in `CovertLayout` constructor as I had initially but it would be a bit inconsistent with the rest of ops. No strong opinion from my side.